bertopic/_bertopic.py

```diff
@@ -3980,18 +3980,29 @@ def _extract_topics(
             embeddings: The document embeddings
             mappings: The mappings from topic to word
             verbose: Whether to log the process of extracting topics
+            calculate_representation: Whether to extract the topic representations
```

Comment:
Shouldn't this be the "additional" topic representations instead since c-TF-IDF is already calculated? I'm struggling a bit with having calculated c-TF-IDF before or not. Now it seems like c-TF-IDF is not calculated at all.

Reply:
Yes, it is. 

The c-TF-IDF is always calculated, since is not conditional to that boolean argument. 
Is calculated in line 3995 `self.c_tf_idf_, words = self._c_tf_idf(documents_per_topic)` and then used to calculate the default representation in `def _extract_words_per_topic`
`if not self.representation_model or not calculate_representation:
            # Default representation: c_tf_idf + top_n_words
            topics = {label: values[: self.top_n_words] for label, values in topics.items()}`